test/Grpc.AspNetCore.Server.Tests/Infrastructure/TestHttpContextAccessor.cs

```diff
@@ -22,6 +22,6 @@ namespace Grpc.AspNetCore.Server.Tests.Infrastructure
 {
     internal class TestHttpContextAccessor : IHttpContextAccessor
     {
-        public HttpContext HttpContext { get; set; }
+        public HttpContext? HttpContext { get; set; }
```

Comment:
I think we can use a constructor for this and make it a non-nullable. There's not much use use otherwise it seems.

Reply:
Sure. It doesn't really matter either way